clusterloader/README.md

```diff
@@ -0,0 +1,79 @@
+# Cluster Loader
+
+Cluster Loader is a tool we use to deploy large numbers of various Kubernetes objects to a cluster. Currently, Cluster Loader has only been tested against local clusters.
+
+## Building
+In order to vendor in e2e/framework & deps we have used [glide](https://github.com/Masterminds/glide).
+After installing glide just run:
+```
+cd perf-tests/clusterloader/
+glide install -v
+go test -c -o e2e.test
+```
+
+## Running
+Cluster Loader uses e2e/framework as a library so the execution may be familiar. This command is used to test against a local cluster:
+```
+./e2e.test --host="127.0.0.1:8080" --ginkgo.v=true --ginkgo.focus="Cluster Loader" --kubeconifg=<path to your kubeconfig> --viper-config=config/test
+```
+`viper-config` does not need the extension of the config file, viper will automatically detect it.
+
+
+## Config
+
+The config file is a basic yaml file that follows the following structure:
+
+```
+# e2e specific vars
```

Comment:
I think it would be clearer if it were a template, i.e. if you use `<something>` in places which user has to provide.